.github/workflows/ci.yml

```diff
@@ -125,7 +126,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-22.04, ubuntu-24.04, macos-14]
+        os: [ubuntu-22.04, ubuntu-24.04, macos-14, ubuntu-24.04-arm] # NOTE: add `ubuntu-22.04-arm, ` to the list
+        # once we have a what4 solvers distribution for Ubuntu 22.04 ARM
```

Comment:
https://github.com/GaloisInc/what4-solvers/releases/tag/snapshot-20251112

(Note that this will require migrating from `macos-14` to `macos-15`.)